tutorials/ngsi_v2/e5_iot_thermal_zone_control/e5_iot_thermal_zone_control_solution.py

```diff
@@ -340,53 +349,57 @@ def on_measurement(client, obj, msg):
     mqttc.disconnect()
 
     print(cbc.get_entity(entity_id=heater.entity_name,
-                         entity_type=heater.entity_type).json(indent=2))
+                         entity_type=heater.entity_type).model_dump_json(indent=2))
 
     # plot results
     fig, ax = plt.subplots()
-    t_simulation = [item["simtime"] for item in history_weather_station]
+    t_simulation = [item["sim_time"]/60 for item in history_weather_station]
```

Comment:
Here and in the plots below I changed the plot x-axis to hours because for some reason the x-axis is here shown differently than in the previous exercise. This exercise does not work as of right now and I am assuming that is the reason why plotting also looks differently on the x-axis.